bin/gwdetchar-scattering

```diff
@@ -164,7 +176,7 @@ if args.state_flag is not None:
                  % (len(statea), args.state_flag, livetime))
 else:
     statea = SegmentList([span])
-
+                   
```

Comment:
@siddharth101, I think this whitespace got put here by mistake, it should be removed and replaced with just a blank line.